bundles/org.openhab.binding.homematic/README.md

```diff
@@ -694,20 +694,28 @@ In scripts:
 
 :::: tabs
 
+::: tab DSL
+
+```php
```

Comment:
shouldn't this be java, not php?

Reply:
yup! Ref: https://github.com/openhab/openhab-docs/issues/999